src/mechanics/integrands/DynamicMomentumBalance.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "mechanics/integrands/MomentumBalance.h"
+
+namespace NuTo
+{
+namespace Integrands
+{
+
+template <int TDim>
+class DynamicMomentumBalance : public MomentumBalance<TDim>
+{
+public:
+    DynamicMomentumBalance(DofType dofType, const Laws::MechanicsInterface<TDim>& law, double rho)
+        : MomentumBalance<TDim>(dofType, law)
+        , mRho(rho)
+    {
+    }
+
+    DofMatrix<double> Hessian2(const CellIpData& cellIpData)
+    {
+
+        NMatrix N = cellIpData.N(MomentumBalance<TDim>::mDofType);
```

Comment:
Just for readability/shortness, I would probably add a line like `auto dof = MomentumBalance<TDim>::mDof`. Or, maybe better, `this->mDof`. No need to change that though.

Reply:
Or `using parent = MomentumBalance<TDim>;` + `parent::mDofType`.